_test_unstructured_client/integration/test_decorators.py

```diff
@@ -140,10 +145,9 @@ def test_integration_split_pdf_with_caching(
     parameters = shared.PartitionParameters(
         files=files,
         strategy=strategy,
-        languages=["eng"],
         split_pdf_page=True,
         split_pdf_cache_tmp_data=use_caching,
-        split_pdf_cache_dir=cache_dir,
```

Comment:
This param doesn't exist, but the test succeeded despite that. Fixed the param anyway.